features/Light Limit Fix/Shaders/LightLimitFix/LightLimitFix.hlsli

```diff
@@ -47,30 +45,38 @@ bool GetClusterIndex(in float2 uv, in float z, out uint clusterIndex)
 bool IsSaturated(float value) { return value == saturate(value); }
 bool IsSaturated(float2 value) { return IsSaturated(value.x) && IsSaturated(value.y); }
 
-float ContactShadows(float3 rayPos, float2 texcoord, float offset, float3 lightDirectionVS, float radius, bool longShadow, uint a_eyeIndex = 0)
+float ContactShadows(float3 viewPosition, float noise, uint3 seed, float3 lightDirectionVS, uint contactShadowSteps, uint a_eyeIndex = 0)
 {
-	float2 depthDeltaMult;
-	uint loopMax;
-	if (longShadow) {
-		loopMax = log2(radius);
-		depthDeltaMult = float2(1.0 * loopMax, 1.0 / loopMax) * 0.10;
-		lightDirectionVS *= loopMax * 4.0;
-	} else {
-		loopMax = log2(radius) * 0.5;
-		depthDeltaMult = float2(1.0 * loopMax, 1.0 / loopMax) * 0.20;
-		lightDirectionVS *= loopMax;
-	}
+	if (contactShadowSteps == 0)
+		return 1.0;
+
+	float2 depthDeltaMult = float2(1.0, 0.05);
+
+	lightDirectionVS *= 2.0;
 
 	// Offset starting position with interleaved gradient noise
-	rayPos += lightDirectionVS * offset;
+	viewPosition += lightDirectionVS * noise;
```

Comment:
`noise` arguments can probably be merged into `viewPosition`, since it is only used here

Reply:
No, because otherwise the arguments are unclear.

Reply:
call is startPositionVS or something